backend/src/main/java/corea/member/domain/MemberRole.java

```diff
@@ -0,0 +1,10 @@
+package corea.member.domain;
+
+public enum MemberRole {
+
+    REVIEWER, REVIEWEE;
+
+    public boolean isReviwer() {
```

Comment:
`ENUM` 도입 너무 좋습니다~ 
리뷰어로만 참여하는 기능에서도 사용할 것 같아요 아마 `BOTH` 같은 값 하나 추가해서? (빠른 머지가 필요할 듯)

```suggestion
    public boolean isReviewer() {
```

Reply:
이거 `[BE] 엔티티 생성 시간 추가(#431)` 여기 pr에서 같이 수정할게요!